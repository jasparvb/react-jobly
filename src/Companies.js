import React from "react";
import Search from './Search';
import CompanyCard from './CompanyCard';

function Companies({companies}) {
    return (
        <div className="Companies col-md-8 offset-md-2">
            <Search search={search} />
            {companies.map(c => 
                <CompanyCard 
                    key={c.handle} 
                    name={c.name} 
                    description={c.description} 
                    logo_url={c.logo_url} 
                />
            )}
        </div>
    );
}

export default Companies;