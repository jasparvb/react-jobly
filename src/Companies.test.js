import React from "react";
import { render } from "@testing-library/react";
import Companies from "./Companies";

it("renders without crashing", function() {
  render(<Companies />);
});

it("matches snapshot", function() {
  const { asFragment } = render(<Companies />);
  expect(asFragment()).toMatchSnapshot();
});
