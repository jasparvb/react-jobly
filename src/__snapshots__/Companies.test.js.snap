// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Companies col-md-8 offset-md-2"
  >
    <div
      class="Search mb-4"
    >
      <form
        class="form-inline"
      >
        <input
          class="form-control form-control-lg flex-grow-1"
          name="search"
          placeholder="Enter search term.."
          value=""
        />
        <button
          class="btn btn-lg btn-primary"
          type="submit"
        >
          Submit
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
